### PR TITLE
[AGGREGATE] minor tweaks / cleanups

### PR DESCRIPTION
**Pull request by csharrison**
This PR makes a few adjustments to the document:

- Replaces `insecure-single-server` with `single-server`, which is broader term that could encompass different notions of security / trust.
- Removes some language about "experimentation" which is not as appropriate for an explainer document. Pushes some of this to the future work section

### CODE REVIEW

File: AGGREGATE.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
   - [Data processing through the aggregation service](#data-processing-through-the-aggregation-service)
     - [High level two-party flow](#high-level-two-party-flow)
```
Comment (csharrison) [inline AGGREGATE.md:18]: Update TOC

**Comment by csharrison** [comment]
@mvale can you review? For some reason can't add you as a formal reviewer

**Comment by mvale** [comment]
LGTM

File: AGGREGATE.md [hunk 2]
```diff
@@ -15,9 +15,9 @@
     - [Aggregate attribution reports](#aggregate-attribution-reports)
     - [Privacy budgeting](#privacy-budgeting)
   - [Data processing through the aggregation service](#data-processing-through-the-aggregation-service)
-    - [High level two-party flow](#high-level-two-party-flow)
+    - [High level two-party flow](#two-party-flow)
```
Comment (johnivdel) [inline AGGREGATE.md:18]: Should this just be Two-party flow to match the heading?

**Comment by johnivdel** [comment]
LGTM

File: AGGREGATE.md (hunk 2 above)
Comment (csharrison) [reply AGGREGATE.md:18]: Yeah let's change that

**Pull request merged**